dev/PushNotifications/PushNotificationReceivedEventArgs.cpp

```diff
@@ -65,9 +65,10 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         return { payload, payload + (length * sizeof(uint8_t)) };
     }
 
-    std::vector<uint8_t> PushNotificationReceivedEventArgs::BuildPayload(std::wstring const& payload)
+    std::vector<uint8_t> PushNotificationReceivedEventArgs::BuildPayload(winrt::hstring const& payload)
     {
-        std::string payloadToSimpleString{ ::winrt::Microsoft::Windows::PushNotifications::Helpers::WideStringToUtf8String(payload) };
+        const std::wstring payloadWideString(payload);
```

Comment:
>const std::wstring payloadWideString(payload) [](http://example.com/codeflow?start=8&length=45)

Maybe WideStringToUtf8String should accept std::wstring_view, which both std::wstring and winrt::hstring convert to.

Reply:
Thats a good point. Let me give that a shot.

Reply:
It turns out that there was just one other reference to this API. I went ahead and changed it to accept hstring only and avoid all conversions altogether. Its much better now!